docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -748,6 +751,25 @@ and rename operations, including in nested structures.
 Table partitioning can also be changed and the connector can still
 query data created before the partitioning change.
 
+Register table
+--------------
+The connector can register table into the metastore with existing metadata and data files.
```

Comment:
Iceberg calls them catalogs, since it's not necessarily a metastore.
```suggestion
The connector can register existing Iceberg tables with the catalog.
```

Reply:
Modified